docs/gadget-devel/building.md

```diff
@@ -16,7 +16,7 @@ Usage:
 Flags:
       --btfgen                  Enable btfgen
       --btfhub-archive string   Path to the location of the btfhub-archive files
-      --builder-image string    Builder image to use (default "ghcr.io/inspektor-gadget/ebpf-builder:latest")
+      --builder-image string    Builder image to use (default "ghcr.io/inspektor-gadget/ebpf-builder:main")
```

Comment:
Should it be `%IG_TAG%` or `%IG_BRANCH%`?

Ditto in other occurrences in the documentation.